entity/contents/solid_stroke_contents.cc

```diff
@@ -143,7 +143,7 @@ bool SolidStrokeContents::Render(const ContentContext& renderer,
                    entity.GetTransformation();
 
   VS::StrokeInfo stroke_info;
-  stroke_info.color = color_;
+  stroke_info.color = color_.Premultiply();
```

Comment:
I think this is an orthogonal concern. One that should be handled by the users of the entity framework. WDYT?

Reply:
Sure, I think it makes sense to bubble up this concern.

Reply:
Done.